crates/biome_css_parser/tests/css_test_suite/ok/scss/value/url.scss

```
@@ -2,23 +2,22 @@ div {
   foo: url(bloo/blah.css);
   bar: url(http://foo/bar/hux.css);
   // TODO(interpolation support): foo: url(fudge#{$x}.css);
-  // TODO(interpolation support): bar: url("http://fudge#{$x}/styles.css");
+  bar: url("http://fudge#{$x}/styles.css");
   // TODO(interpolation support): hux: url(http://box_#{$y}////fudge#{$x}.css);
```

Stylelint (17.6.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)